bin/apps.bash

```diff
@@ -24,15 +24,15 @@ apps_run_sc() {
     log_info "Applying applications in service cluster"
     (
         with_kubeconfig "${config[kube_config_sc]}" \
-            "${scripts_path}/deploy-sc.sh" "${1:-""}"
+            "${scripts_path}/deploy-sc.sh" "${1:-""}" "$2"
```

Comment:
This made me a bit suspicious. Since `$1` seems to be optional (because it has a fallback on `""`) I thought we shouldn't be able to assume that `$2` is set either. But it looks like the only uses of these functions are making sure to always set '$1`:
```console
$ fgrep -r 'apps_run_sc'
bin/apps.bash:apps_run_sc() {
bin/apps.bash:    apps_run_sc "${2:-""}" "$3"
$ fgrep -r 'apps_run_wc'
bin/apps.bash:apps_run_wc() {
bin/apps.bash:    apps_run_wc "${2:-""}" "$3"
```

Maybe just make it easy for ourselves and pass on all arguments?
```suggestion
            "${scripts_path}/deploy-sc.sh" "${@}"
```